bin/wasm-node/javascript/smoldot-demo-db.json

```diff
@@ -0,0 +1 @@
6121d58b930a48f485dde4be2c8f339c97615","weight":1},{"public_key":"486996caf3e195b25a75a0f85d15ece979c2810dc28b52e4cb7ee66a9ecb143d","weight":1},{"public_key":"f4ac905b0f5600db59569a85c86a31f99cd4d14931709daf5f4a6f948806964a","weight":1},{"public_key":"36a39760f4daf035d5a338a33cc4919f13a7799db697ae5a1baf35d4b500995f","weight":1},{"public_key":"e2e048aeb5ececfb215b3940441d39a303b3f02e8f1fdd6c80a395b95c82ac06","weight":1},{"public_key":"78f7239b1d3fd24b2e22163d944288723be90f5c3ab491260f82ef56b8ee2508","weight":1},{"public_key":"361a4f7e4a840c1a325be4018cf53ed2e412d8e7b7ea86b10c2932d658f35d33","weight":1},{"public_key":"b237e7ea16546bf8fb88a5fcda8d49f5952872eaf3fff9f430244b1f1cc38829","weight":1},{"public_key":"e04da7bf37afaed6d25f7686c59360d65a40e90bcf70488ae605831172f1f005","weight":1},{"public_key":"aee49741653bd7527e93760c31f864147e291049a20ddffd22cecfb461855d0c","weight":1},{"public_key":"c602026c5f408bacf526cac19912e84570756869bea351d5b829bc7cce9f9e65","weight":1},{"public_key":"863ec9619078a15a5a4c6fe3ac05e623b2c03bf32a95f33ed7af1c54c0f30753","weight":1},{"public_key":"64da80a836412786a6f5176f4378edeb60c1a337e6c8490a235044e287fc4144","weight":1},{"public_key":"06e727ff454a8e503af3ae2f7ffb1348ef7cacc75021e8d333e2b4ec83269244","weight":1},{"public_key":"0489716b1767d671b5b4b254f0d6f6acc0d8fd96c8e72dbbf2d7ece068245e56","weight":1}],"randomness":"5444935ba3688f95f9e7ecc5b2b130e05a3432271ad62687198b8d18e2110782","c":{"num":1,"denom":4},"allowed_slots":"primary-and-secondary-plain"},"grandpa_after_finalized_block_authorities_set_id":3186,"grandpa_finalized_triggered_authorities":[{"public_key":"1154ce4d645bca3bc43a485f09ba43fd8f8deffd5503b144f316773b9bca0639","weight":1},{"public_key":"d6b71db058c2b725d9a1017687cb31bd71c3845e970eaa63bb40ed47b4781cc8","weight":1},{"public_key":"c1c8b7680700472f35c23d539c36c87895724189c2158be4c9d8c818bc0a3dd6","weight":1},{"public_key":"404b31a6663344c68c7e7c2a1cf4a767f53a525fe022f80da283d2c8bb13686d","weight":1},{"public_key":"39fd30cff65229eb5137e7020f5b167d56f4ddc7b94838578a3c2133bb06de3a","weight":1},{"public_key":"5928686fa1be237692bd9896180cc83ee9e58c7b54872446cf72ef18c4caae7e","weight":1},{"public_key":"ca6fda68419e374351c2fc8314b2b1b636932390a23224d23c30197a20ec4cb2","weight":1},{"public_key":"31dae797bbac0e27b901355cc90446369dcd15d8f965933b57723ee389670d55","weight":1},{"public_key":"169da96fe889fe19f2e9463c4cb730b33473a561f0a15d5581ca7c52362a252d","weight":1},{"public_key":"b6f9a5a4b86a88db316afb7e2959804fd6b1e3e5774035e4e0179dc7762b1b63","weight":1},{"public_key":"7cf015bc8c5d14382b396df342b9e14a07507af1f8c96e0532a1b93b70c7166c","weight":1},{"public_key":"98ab15a1b8374e8e6dc53f594d8a6c1c481c71fe1c32fe5b58708e78126c2229","weight":1},{"public_key":"d1226fadc01b5271da540ad505db93e2f855b2b6cc2500f9b3878e63660512d4","weight":1},{"public_key":"07cbf08a347b44d8c3e9e661c02f1a9d1def2a324bad2c50965f6e5907550f04","weight":1},{"public_key":"f79cf74c9dba0d4803234f266947bbc058a1c70541b3f7c3f5db2448d41bd4d2","weight":1}]}
```

Comment:
Please remove

Reply:
Damn it I thought I'd removed that from my commit.

Reply:
Should i add it to .gitignore too?

Reply:
The database isn't working anymore. No code is reading/writing this file anywhere. So no